src/runtime/types.cpp

```diff
@@ -2910,6 +2912,51 @@ extern "C" PyUnicodeObject* _PyUnicode_New(Py_ssize_t length) noexcept {
     return unicode;
 }
 
+static void setTypeGCProxy(BoxedClass* cls) {
```

Comment:
comments as to the purpose of these two functions (particularly `setTypeGCNone` would be useful)